kolko.py

```diff
@@ -0,0 +1,66 @@
+
+def printer(tab):
+    'print table with positions to choose'    
+    print('{}\n{}\n{}'.format(tab[0],tab[1],tab[2]))
+
+def instert_to_tab(player,n):
+    '''inserting player to table return update table
+        player - X or O
+        n - position number'''    
+    c=0
```

Comment:
Nazywanie zmiennych pojedynczymi literami zwykle jest złym pomysłem. Nie mówi to osobie która czyta zbyt wiele co ta zmienna reprezentuje.

Reply:
Proponuje tutaj zamienić c na `counter`

Reply:
Tak przy okazji w większości miejsc pozamieniaj tego typu konstrukcje: `c=0` na `c = 0`. Taki jest generalny standard jakości kodu.

Reply:
wynika to  PEP 8 ??